UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeSetUtilities.java

```diff
@@ -270,15 +273,20 @@ private boolean applyPropertyAlias0(
                         if (invert != UnicodeProperty.equals(i, prop.getValue(i))) {
                             set.add(i);
                         }
+                        invert = false;
```

Comment:
please move this to after the loop. surely we need not do this 1.1M times...

Reply:
😂 At least we are sure that it will be _very_ false !